resources/build/advanced/fluree-crypto.js

```diff
@@ -0,0 +1,476 @@
+if(typeof Math.imul == "undefined" || (Math.imul(0xffffffff,5) == 0)) {
```

Comment:
Should this build output be committed? I think all other things equal I would prefer compiling artifacts during a release process each time instead of checking build artifacts into versio control.

Reply:
All cleaned out per your comment @zonotope and ready for further review if needed.